nakama/cardinal.go

```diff
@@ -50,9 +49,14 @@ func makeURL(resource string) string {
 	return fmt.Sprintf("%s/%s", globalCardinalAddress, resource)
 }
 
-func cardinalListEndpoints(path string) ([]string, error) {
-	url := makeURL(path)
-	resp, err := http.Get(url)
+type endpoints struct {
+	TxEndpoints    []string `json:"tx_endpoints"`
+	QueryEndpoints []string `json:"query_endpoints"`
+}
+
+func cardinalGetEndpointsStruct() (*endpoints, error) {
```

Comment:
maybe just do return type of `(txEndpoints []string, queryEndpoints []string, err error)`

Reply:
@jerargus and I talked about, never using using named return values in the code as a company policy. I agree with him overall, but for this specific case I don't care too much. @jerargus thoughts?

Reply:
Named returns are good. Especially when returning multiple values of the same type. [Naked returns](https://levelup.gitconnected.com/go-naked-returns-4e2094b598e6) are bad.

Reply:
Oh my mistake.